### PR TITLE
Update TestCommon and remove deprecated FxCopAnalyzers

### PR DESCRIPTION
**Pull request by HenrikSommer**
## Description

Minor changes to code-analyzing and integration testing:
* Deprecated FxCopAnalyzers is removed. It is superseded by .Net Analyzers
* Energinet.DataHub.Core.FunctionApp.TestCommon updated to 1.2.0

## References

* #0000

### CODE REVIEW

File: source/Energinet.Charges.Libraries/source/Directory.Build.props [hunk 1]
```diff
@@ -16,10 +16,6 @@
             <PrivateAssets>all</PrivateAssets>
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
-        <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="3.3.1">
```
Comment (BjarkeMeier) [inline source/Energinet.Charges.Libraries/source/Directory.Build.props:19]: Why do we have a `Directory.Build.props` file in libraries but not in charges solution? Are we missing some alignment?

Comment (BjarkeMeier) [inline source/Energinet.Charges.Libraries/source/Directory.Build.props:19]: Should this be replaced by .Net Analyzers?

Comment (HenrikSommer) [reply source/Energinet.Charges.Libraries/source/Directory.Build.props:19]: We do have a general `Directory.Build.props`, it is placed in the root of `geh-charges`

Comment (BjarkeMeier) [reply source/Energinet.Charges.Libraries/source/Directory.Build.props:19]: Then why don't we delete this one?
And perhaps move the one from the root into the `source/` folder?

Comment (HenrikSommer) [reply source/Energinet.Charges.Libraries/source/Directory.Build.props:19]: .Net Analyzers are enabled by default in .Net 5.0 projects according to [this guilde](https://docs.microsoft.com/da-dk/visualstudio/code-quality/install-net-analyzers?view=vs-2022)

Comment (HenrikSommer) [reply source/Energinet.Charges.Libraries/source/Directory.Build.props:19]: I agree, resolved in 7084d5fe

**Review by bemwen — APPROVED** [review]

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**